### DISCUSSION TITLE
ScriptReference/EditorPrefs

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#197] [Notes]
# ScriptReference/EditorPrefs

https://docs.unity3d.com/ScriptReference/EditorPrefs.html

**Comment by mvi** [comment]
Note that on Windows the registry location is  
  
`HKCU\Software\Unity Technologies\Unity Editor N.x`  
rather than the documented  
  
`HKCU\Software\Unity Technologies\UnityEditor N.x`

**Comment by pavelminyaev** [comment]
Worth noting, this is NOT per project. So if you have multiple Unity projects on your PC, they will share the prefs.
To `EditorPrefs` per project, just add some project-specific prefix to your prefs keys.
